tests/fixtures/posts_in_subfolder/_layouts/default.liquid

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <!-- start default.liquid -->
+  <head>
+    {% if is_post %}
+      <title>{{ title }}</title>
+    {% else %}
+       <title>Cobalt.rs Blog</title>
+    {% endif %}
+  </head>
+  <body>
+    <div>
+      {% if is_post %}
+        {% include '_layouts/post.liquid' %}
+      {% else %}
+        {{ content }}
+      {% endif %}
+    </div>
+  </body>
+  <!-- start default.liquid -->
```

Comment:
That fix changes all html files in the directory tests/target/posts_in_subfolder too.